src/components/CartContextWrapper/CartContextWrapper.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
+
+import { CartContext } from "../../util/context";
+
+export const CartContextWrapper = ({ children }) => {
+  const [cart, setCart] = useState([
```

Comment:
Let's eliminate the initial cart items so that we don't have to remove these later on.